web/src/server_events_dispatch.js

```diff
@@ -100,11 +100,20 @@ export function dispatch_normal_event(event) {
             attachments_ui.update_attachments(event);
             break;
 
-        case "custom_profile_fields":
+        case "custom_profile_fields": {
             realm.custom_profile_fields = event.fields;
             settings_profile_fields.populate_profile_fields(realm.custom_profile_fields);
             settings_account.add_custom_profile_fields_to_settings();
+            const $navbar_alert = $("#navbar_alerts_wrapper").children(".alert").first();
+            if (
+                !$navbar_alert?.length ||
+                $navbar_alert.is("#empty-required-profile-fields-warning") ||
+                $navbar_alert.is(":hidden")
+            ) {
+                navbar_alerts.maybe_show_empty_required_profile_fields_alert();
+            }
             break;
+        }
```

Comment:
I would not expect you to need to add these outer braces here; is there a reason for this inconsistency with the existing style?

Reply:
It was to suppress the linting error when I was defining a variable in case block ```const $navbar_alert = $("#navbar_alerts_wrapper").children(".alert").first();```. It is not needed now though.

Reply:
I see. In this file in particular, generally the goal is to have modules responsible for specific subsystems do anything complicated, just because otherwise this file would be infinitely long anyway.